docs/devtools.md

```diff
@@ -1,33 +1,24 @@
 # Developer Tools
 
-This section describes tooling and features included in Koku to assist
-developers contributing to Koku.
+This section describes tooling and features included in Koku to assist developers contributing to Koku.
 
 ## Environment Variables
 
-Koku makes use of environment variables to configure features and
-application capabilities.
+Koku makes use of environment variables to configure features and application capabilities.
 
-In the repository, there is an `.env.example` file with sample
-environment settings. The `.env` file is used by Django\'s tools as well
-as Koku\'s scripting. It is the recommended way to configure your local
-development environment settings.
+In the repository, there is an `.env.example` file with sample environment settings. The `.env` file is used by Django\'s tools as well as Koku\'s scripting. It is the recommended way to configure your local development environment settings.
```

Comment:
Escaping the single quotes here is craft from the RST conversion and can be removed.
```suggestion
In the repository, there is an `.env.example` file with sample environment settings. The `.env` file is used by Django's tools as well as Koku's scripting. It is the recommended way to configure your local development environment settings.
```